include/aspect/material_model/interface.h

```diff
@@ -869,6 +882,13 @@ namespace aspect
                       const FullMatrix<double>  &/*expansion_matrix*/) override
         {}
 
+
```

Comment:
up in line 878 (cannot comment there) you can now remove the `= 0` from the `get_nth_output` function, because we now have a default implementation.